src/front/styles/Header.css

```diff
@@ -0,0 +1,29 @@
+header {
+  color: white;
+  background: #3da2daff;
+  background-image: url("https://github.com/nannyhub/NannysHub/blob/landingpage/src/front/img/nan.jpg");
```

Comment:
I couldn't see this image in my computer... Could you check if everything is right over here?